jetty-documentation/src/main/asciidoc/administration/sessions/session-configuration-sessioncache.adoc

```diff
@@ -34,6 +34,7 @@ Once the `session-cache-hash` module has been enabled, you can view a list of al
 #jetty.session.saveOnInactiveEvict=false
 #jetty.session.saveOnCreate=false
 #jetty.session.removeUnloadableSessions=false
+#jetty.session.flushOnCommit=false
```

Comment:
Use the new name `flushOnResponseCommit`.

Reply:
Done